hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java

```diff
@@ -59,6 +62,59 @@ public void testEnsureFileCreatedImmediately() throws Exception {
     assertIsFile(fs, TEST_FILE_PATH);
   }
 
+  /**
+   * {@link AbfsOutputStream#incrementWriteOps()}
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testWriteOpsMetric() throws Exception {
```

Comment:
move into the same tests as the reads for ease of backports

Reply:
I'll remove this test and add write operations tests in testAbfsInputStreamReadOps

Reply:
I'll remove this test and add write operations and validating tests in testAbfsInputStreamReadOps(change it's name to testAbfsStreamOps) ?